api/lib/config.js

```diff
@@ -307,7 +307,7 @@ module.exports = (function () {
     config.jwtConfig.livretScolaire = { secret: 'secretosmose', tokenLifespan: '1h' };
     config.jwtConfig.poleEmploi = { secret: 'secretPoleEmploi', tokenLifespan: '1h' };
 
-    config.logging.enabled = false;
+    config.logging.enabled = process.env.LOG_ENABLED === 'true' ? true : false;
```

Comment:
Y'a moyen de simplifier, mais ça cache l'intention que l'on veut le désactiver par défaut
WDYT ?
```suggestion
    config.logging.enabled = process.env.LOG_ENABLED === 'true';
```

Reply:
Pas certain que ca donne plus d'intention avant

Reply:
Voir https://github.com/1024pix/pix/pull/4021#discussion_r799296210